pallets/circuit-breaker/README.md

```diff
@@ -0,0 +1,6 @@
+### Circuit Breaker pallet
+
+This pallet is used to keep track of the initial liquidity of assets (of an AMM), 
+then to validate the next liquidity volume per block per asset,
+
+Before using validation, the initial liquidity for an asset must be registered with the pallet, otherwise it results in error.
```

Comment:
hm.this readme does not make much sense to me.

after reading, i could not tell what it does. I'd prefer to rewrite it.

Reply:
rewritten, please check again if it makes more sense.